debugtools/DDR_VM/src/com/ibm/j9ddr/corereaders/macho/ThreadCommand.java

```diff
@@ -52,55 +52,21 @@
 public class ThreadCommand extends LoadCommand
 {
 
-	// thread flavor values from <mach/i386/thread_status.h>
-	public static final int x86_THREAD_STATE32 = 1;
-	public static final int x86_FLOAT_STATE32 = 2;
-	public static final int x86_EXCEPTION_STATE32 = 3;
-	public static final int x86_THREAD_STATE64 = 4;
-	public static final int x86_FLOAT_STATE64 = 5;
-	public static final int x86_EXCEPTION_STATE64 = 6;
-	public static final int x86_THREAD_STATE = 7;
-	public static final int x86_FLOAT_STATE = 8;
-	public static final int x86_EXCEPTION_STATE = 9;
-	public static final int x86_DEBUG_STATE32 = 10;
-	public static final int x86_DEBUG_STATE64 = 11;
-	public static final int x86_DEBUG_STATE = 12;
-	public static final int THREAD_STATE_NONE = 13;
-	public static final int x86_AVX_STATE32 = 16;
-	public static final int x86_AVX_STATE64 = (x86_AVX_STATE32 + 1);
-	public static final int x86_AVX_STATE = (x86_AVX_STATE32 + 2);
-	public static final int x86_AVX512_STATE32 = 19;
-	public static final int x86_AVX512_STATE64 = (x86_AVX512_STATE32 + 1);
-	public static final int x86_AVX512_STATE =(x86_AVX512_STATE32 + 2);
-
-	public static class ThreadState
+	public static abstract class ThreadState
```

Comment:
Please put modifiers in canonical order (`public abstract static class ThreadState`).

Reply:
Fixed.